### ISSUE TITLE
WeekView/MonthView - Keyboard not working

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#3942]
**Describe the bug**

**To Reproduce**
1. Go to http://localhost:4000/components/week-view/example-index.html
1. Try to tab into the weekview
1. Notice the focus state stalls
1. go to  http://localhost:4000/components/monthview/example-index.html
1. Notice there are a few extra tabs before you hit the calendar

**Expected behavior**
Should be able to key into everything in the week and month view. Maybe see about other keyboard shortcuts like google cal.

**Version**
- ids-enterprise: [e.g. v4.29]

**Platform**
 - Mac/Chrome

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
